planning/robot_model_loader/src/robot_model_loader.cpp

```diff
@@ -51,11 +51,12 @@ robot_model_loader::RobotModelLoader::RobotModelLoader(const Options &opt)
   configure(opt);
 }
 
-robot_model_loader::RobotModelLoader::~RobotModelLoader()
-{
+robot_model_loader::RobotModelLoader::~RobotModelLoader() {
   model_.reset();
   rdf_loader_.reset();
   kinematics_loader_.reset();
+
+  ROS_DEBUG( "Robot model loader destructor" );
```

Comment:
I would prefer not to merge all the debug code.
MoveIt is full of such messages already
and it becomes quite an overhead when reading through the code.

Could you please reduce this request to the functionally important
diff in the destructor?
Could you also add a short explanation to the commit explaining why
this is required?